external/settings.php

```diff
@@ -7,6 +7,12 @@
 
 $tmpl = new OCP\Template( 'external', 'settings');
 
-$tmpl->assign('images', glob(\OC_App::getAppPath('external') . '/img/*.*'));
+$images = glob(\OC_App::getAppPath('external') . '/img/*.*');
```

Comment:
well, technically you didn't introduce this private API call, so I guess it's ok to leave it until there is a way in core